db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/amazon_check_pre_retirement.rb

```diff
@@ -12,7 +12,7 @@
   $evm.log('info', "Instance:<#{vm.name}> on EMS:<#{ems.try(:name)} has Power State:<#{power_state}>")
 
   # If VM is powered off or this instance is running on an instance store
-  if %w(off suspended).include?(power_state) || vm.hardware.root_device_type == "instance_store"
+  if %w(off suspended terminated).include?(power_state) || vm.hardware.root_device_type == "instance_store"
```

Comment:
@durandom Stopping and removing the instance from a provider is only part of the retirement workflow.  There can be other user-defined actions needed for a customer's specific use-case.

The change here is fine if @blomquisg approves of the power_state value change in this PR.